cmd/hack.go

```diff
@@ -12,29 +12,34 @@ import (
   "github.com/spf13/cobra"
 )
 
-var abortFlag bool
-var continueFlag bool
+type HackFlags struct {
```

Comment:
Most of our commands have an abort / continue but actually not all of them. Only those that have the potential to hit conflicts should really have them. `repo`, `prune-branches`, `rename-branch`, and `kill` all don't need them.